tsconfig-base.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "lib": ["WebWorker", "DOM", "DOM.Iterable"],
+    "lib": ["WebWorker", "DOM", "DOM.Iterable", "ES2022"],
```

Comment:
```suggestion
    "lib": ["WebWorker", "DOM", "DOM.Iterable"],
```
What was this intended to do?  @ShaunSHamilton

Reply:
I cannot remember. Potentially, it was needed for one of the newer Object/Array methods - normally why I add it to configs.

That is to say, it may have been necessary for a change that is **no longer** here.

Reply:
No worries.  Let's try without and see if any type errors appear.